src/components/CreateProfile/CreateProfile.jsx

```diff
@@ -116,10 +123,6 @@ const ProfilePage = ({ classes }) => {
 
   const handleSubmit = (e, updatedInfo) => {
     e.preventDefault();
-    // Needs to make a post call to backend to submit profileinfo state as updated user details
-    // Remember to trim( empty spaces from the end of strings)
-    // eslint-disable-next-line no-console
-    console.log(updatedInfo, user);
```

Comment:
I realized that we can't actually do this. Async doesn't seem to wait in this function. I had to change this in the `AddReview` flow https://github.com/outintech/nbjc-webapp/blob/main/src/routes/AddReview/AddReview.jsx#L95-L119